aeat.py

```diff
@@ -1064,6 +1062,9 @@ def get_issued_sii_reports(cls):
             issued_invoices[company]['A1'] = periods2
 
             today = date.today()
+            invoice_date = today
+            if config.sii_default_offset_days:
+                invoice_date = today - timedelta(days=config.sii_default_offset_days)
```

Comment:
It's more Pythonic do:

invoice_date = (today - timedelta(days=config.sii_default_offset_days)
    if config.sii_default_offset_days else today)

Or the same you have done some lines below:

                    report.load_date = (date.today() -
                            timedelta(days=config.sii_default_offset_days
                                      if config.sii_default_offset_days else 0))